pkg/http/request.go

```diff
@@ -0,0 +1,68 @@
+package http
+
+import (
+	"net/http"
+
+	"github.com/google/uuid"
+	"github.com/grafana/sobek"
+	"go.k6.io/k6/js/modules"
+
+	"github.com/grafana/xk6-http/internal/interfaces"
+)
+
+// Request object
+type Request struct {
```

Comment:
This is a key component we have to comment heavily. For example, what is an ID for a request?